src/modules/fancyzones/editor/FancyZonesEditor/DashCaseNamingPolicy.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Copyright (c) Microsoft Corporation
+// The Microsoft Corporation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Text.Json;
+
+using FancyZonesEditor.Utils;
+
+namespace FancyZonesEditor
+{
+    public class DashCaseNamingPolicy : JsonNamingPolicy
+    {
+        public static DashCaseNamingPolicy Instance { get; } = new DashCaseNamingPolicy();
+
+        public override string ConvertName(string name)
+        {
+            // Conversion to other naming conventaion goes here. Like SnakeCase, KebabCase etc.
```

Comment:
This is conversion **from** other case styles (camel, snake, kebab etc.) to dash case, comment is bit misleading.

Reply:
Comment removed